docroot/themes/custom/twentyeighteen/sass/components/hero/_hero.scss

```diff
@@ -0,0 +1,26 @@
+// Hero
+//
+// Hero Styling
+//
+// Markup: hero.twig
+//
+// Style guide: components.hero
+
+.hero-block {
+  width: 100%;
+}
+
+.hero-background-img {
+  position: relative;
+}
+
+.hero-logo {
+  display: none;
```

Comment:
Why are we doing display: none for smaller widths?

Reply:
@lkopacz from the mocks in https://drupal4gov.atlassian.net/browse/DGC-3 there is a logo image floating in the middle of the background image on larger screen sizes. On the small width mock, the logo is gone so I hid it with display: none. Is there a better way to do this?